### ISSUE TITLE
can't pass arguments to `@energy`

### ISSUE THREAD

**Issue by mdav2** [FermiQC/Fermi.jl#28]
We should be able to do something like
```
@set basis cc-pvdz
wf1 = @energy rhf
wf2 = @energy rhf <- wf1
```
to read in starting wfs.

**Comment by mdav2** [comment]
This should work for CC and CI as well as HF.

**Comment by gustavojra** [comment]
Waiting #40 to solve this one

**Comment by gustavojra** [comment]
Fixed #39
